<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>641</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>50</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : rgb(55, 57, 63);
color: rgb(229, 229, 229);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="bemVindo">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>520</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mercado</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(240, 240, 240);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Mercado</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>O que deseja fazer ?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_estoque">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(81, 85, 94);</string>
    </property>
    <property name="text">
     <string>Editar Materiais</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_venda">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>310</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(81, 85, 94);</string>
    </property>
    <property name="text">
     <string>Remover Material</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_produtos">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(81, 85, 94);</string>
    </property>
    <property name="text">
     <string>Listar Material</string>
    </property>
   </widget>
   <widget class="QPushButton" name="material">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>370</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(88, 92, 102);</string>
    </property>
    <property name="text">
     <string>Adicionar Novo Material</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logout">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>104</width>
      <height>46</height>
     </rect>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(81, 85, 94);</string>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>834</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
